websites.yml

```diff
@@ -144,3 +144,6 @@ avatar:
   tumblr: opengraph
   twitter:
     url: https://twitter.com/{u}/profile_image?size=original
+logos:
+  keybase:
+    url: https://png.icons8.com/ios/30/000000/keybase.png
```

Comment:
Should add an example for font awesome usage.

Reply:
Do we have a website which needs the `fontawesome:` key?

If not, cant be added, but it looks like it should work, and the next student can do the testing or fix it if necessary.